modules/localgov_workflows_notifications/README.md

```diff
@@ -0,0 +1,70 @@
+# Localgov Workflow Notifications
+
+This module is designed to send notifications to service contacts when content moderation
+states change. It currently only implements email notifications when content passes its needs
+review date, but it should be easy to implement email notifications on other events.
+
+## Setup
+
+The server must be able to send emails and cron must be running frequently.
+
+For email sending to work you'll need to set the default Symfony Mailer transport for your
+environment here: /admin/config/system/mailer/transport. In the settings there's also an
+option to send test emails so you can check your settings work.
+
+Emails are sent when either Drupal cron is run or the `localgov_workflows_notifications_email`
+queue is processed. It is recommended that Drupal cron runs at least every 10 minutes and uses
+an external trigger to run. For more information on setting up an external cron job see
+<https://www.drupal.org/node/23714>.
+
+## Structure
+
+There are 3 main components to the module; service contact management, notification creation
+and email sending.
+
+### Service contacts
+
+The module defines a service contact entity. A service contact is either an existing Drupal
+user or an external user with a name and email address.
+
+One or more service contacts can be associated with a piece of content. In theory this can
+be any content entity, but the widget to associate service contacts with content currently
+only works with nodes.
+
+### Notifications
+
+Any notifications added to the email queue will be sent at the next cron run.
+
+Needs review notifications are sent every X days for content with a needs review date that
```

Comment:
>  "drupal/symfony_mailer": "^1.4",

This is the Drupal Symfony Mailer module (https://www.drupal.org/project/symfony_mailer) rather than the Symfony library that is now included with Drupal core.

> "every X days" - should this wording remain as it is now?

This number of days is configurable, but the wording can certainly be improved.